spdx-utils/src/main/kotlin/SpdxLicenseAliasMapping.kt

```diff
@@ -131,6 +131,7 @@ object SpdxLicenseAliasMapping {
         "wtf" to WTFPL,
         "wtfpl" to WTFPL,
         "unicode/icu" to ICU,
+        "x11" to X11,
```

Comment:
If at all, this should go to the previous commit. But again, as this only differs in case from "X11" it should not be needed at all.

Reply:
Fixed